### PR TITLE
implement SAIC-50: Add support for dry running CF

### PR DESCRIPTION
**Pull request by oleksii-shyman**
- added "dry_run" task in CloudFerry that runs stub tasks
  - created stub tasks

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/74/

File: dry_run/tasks/simple.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+from cloudferrylib.scheduler import task
+from cloudferrylib.utils import utils as utl
```
Comment (roman-verchikov) [inline dry_run/tasks/simple.py]: Very minor, but what is the reason of saving 2 bytes here? :) Just leave it as `from cloudferrylib.utils import utils as utl`, "utils" is much more readable than "utl".

File: dry_run/tasks/simple.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+LOG = utl.get_log(__name__)
+
+
+class AddNumbers(task.Task):
+
+    def __init__(self, a, b):
+        self.a = a
+        self.b = b
+        super(task.Task, self).__init__()
```
Comment (roman-verchikov) [inline dry_run/tasks/simple.py]: Please move the `super()` call to the top, you should not rely on what base class is doing.

File: dry_run/tasks/simple.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        LOG.debug("adding '%d' to '%d'", self.a, self.b)
+        return {'res1': self.a + self.b}
+
+
+class DivideNumbers(task.Task):
+
+    def __init__(self, a, b):
+        self.a = a
+        self.b = b
+        super(task.Task, self).__init__()
```
Comment (roman-verchikov) [inline dry_run/tasks/simple.py]: ditto

File: dry_run/tasks/simple.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        LOG.debug("dividing '%d' by '%d'", self.a, self.b)
+        return {'res2': self.a / self.b}
+
+
+class MultiplyNumbers(task.Task):
+
+    def __init__(self, a, b):
+        self.a = a
+        self.b = b
+        super(task.Task, self).__init__()
```
Comment (roman-verchikov) [inline dry_run/tasks/simple.py]: ditto

**Comment by roman-verchikov** [comment]
Looks good other than few minor comments inline. Please fix.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/76/

**Comment by roman-verchikov** [comment]
Looks good. :+1:

**Comment by miarmak** [comment]
For me either.

**Pull request merged**